### PR TITLE
CB-15404 Introduce Cloudera Runtime 7.2.15

### PR DESCRIPTION
**Pull request by pkedvessy**
See detailed description in the commit message.

### CODE REVIEW

**Comment by horadla23** [comment]
why do we have a different PR for 2.51 and master?

File: datalake/src/main/resources/application.yml [hunk 1]
```diff
@@ -111,8 +111,8 @@ datalake:
       ssl: false
   runtimes:
     default: "7.2.13"
-    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
-    supported: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.14"
+    advertised: "7.1.0,7.2.0,7.2.1,7.2.2,7.2.6,7.2.7,7.2.8,7.2.9,7.2.10,7.2.11,7.2.12,7.2.13,7.2.147.2.15"
```
Comment (horadla23) [inline datalake/src/main/resources/application.yml:114]: `'` is missing here as well at the end of the line

**Review by horadla23 — CHANGES_REQUESTED** [review]

File: datalake/src/main/resources/application.yml (hunk 1 above)
Comment (pkedvessy) [reply datalake/src/main/resources/application.yml]: Fixed here as well. Thanks

**Review by horadla23 — APPROVED** [review]

**Comment by pkedvessy** [comment]
retest this please

**Review by doktoric — APPROVED** [review]

**Pull request merged**